pyproject.toml

```diff
@@ -167,6 +161,12 @@ crawler = [
   "webdriver-manager",
 ]
 preprocessing = [
+  "nltk",
+  "langdetect",  # for language classification
+]
+file-conversion = [
+  "python-docx",
```

Comment:
As far as I know dC supports docx files so they will want to install file-conversion either in the base images directly or after. Could you please sync with @ArzelaAscoIi ? https://github.com/deepset-ai/haystack/blob/bc95de5dc114b1c60343ed9b71d9bb639ed1dddb/docker/docker-bake.hcl#L48

Reply:
So far we don't include docx :) All good. Thanks for asking!